packages/magento-cart-items/components/SelectedCustomizableOptions/SelectedCustomizableOptions.tsx

```diff
@@ -19,15 +19,30 @@ export function SelectedCustomizableOptions(props: ConfigurableActionCartItemPro
   return (
     <>
       {options.map((option) => (
-        <Typography variant='body2' component='div' key={option.customizable_option_uid}>
-          <Typography variant='subtitle2' component='span'>
+        <Typography
+          variant='body2'
+          component='div'
+          key={option.customizable_option_uid}
+          sx={(theme) => ({ display: 'flex', gap: theme.spacings.xxs })}
+        >
+          <Typography variant='subtitle2' component='span' sx={{ color: 'text.primary' }}>
             {option.label}
           </Typography>
           {option.values.filter(nonNullable).map((value) => (
-            <span key={value.customizable_option_value_uid}>
-              {value.price.value > 0 && <Money value={value.price.value} />} {value?.label}
-              {value?.value}
-            </span>
+            <>
+              <span key={value.customizable_option_value_uid}>
+                {value.price.value > 0 && <Money value={value.price.value} />} {value?.label}
+              </span>
+              <Box
+                sx={(theme) => ({
+                  [theme.breakpoints.down('sm')]: {
+                    display: 'none',
+                  },
+                })}
+              >
+                {value.value}
```

Comment:
I don't think it is a good idea to hide stuff on mobile, can't we solve this in a different way? If information is important, it is also important on mobile